tests/source/comment_multiline_indentation.rs

```diff
@@ -0,0 +1,46 @@
+// Ensure multiline comments are indented properly,
+// incluyding when second line is prefixed by tab or at the beginning of the line
```

Comment:
typo fix :smile: also, can you add a copy of this test case but with the hard_tabs set to true? I wouldn't expect any issues, but good to have for regression regardless

```suggestion
// including when second line is prefixed by tab or at the beginning of the line
```

Reply:
Fixed and added `...-hard-tabs.rs` test file.